apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -444,7 +447,11 @@ namespace MWPhysics
                     const btCollisionObject* standingOn = tracer.mHitObject;
                     PtrHolder* ptrHolder = static_cast<PtrHolder*>(standingOn->getUserPointer());
                     if (ptrHolder)
+                    {
+                        mutex.lock();
                         standingCollisionTracker[ptr] = ptrHolder->getPtr();
+                        mutex.unlock();
```

Comment:
It is always better to use `lock_guard` to guarantee that the mutex will be always released, even when it seems that no exceptions will exist. C++ has long-range action, changes in one part of the code can easily affect another